tmt/utils.py

```diff
@@ -6183,3 +6190,33 @@ def __exit__(self, *args: Any) -> None:
     @property
     def duration(self) -> datetime.timedelta:
         return self.end_time - self.start_time
+
+
+def retry(
+        func: Callable[..., T],
+        retries: int,
+        delay: int,
+        label: str,
+        logger: tmt.log.Logger,
+        *args: Optional[Any],
+        **kwargs: Optional[Any]
```

Comment:
Both `Optional` are not needed. `Any` already covers `None`, it's really *any* type.